### DISCUSSION TITLE
cuda_version

### DISCUSSION THREAD

**Discussion by georgeper2001** [azavea/raster-vision#2115] [General]
I am going to build a new docker image using the Dockerfile (which is in the tests folder in raster-vision) to update to v0.20 of rastervision and the environment. I have a nvidia rtx 3080 card in the ubuntu computer i will be using to create the model.  the arguments, ARG's, that the script takes are the following:
ARG UBUNTU_VERSION=20.04
ARG CUDA_VERSION
ARG PYTHON_VERSION=3.9
ARG TARGETPLATFORM
ARG TARGETARCH

for TARGETPLATFORM i am going to set it at "linux/arm64"
and for TARGETARCH i am going to set it at "arm64"

what should i use for CUDA_VERSION?

**Comment by AdeelH** [comment]
- There is no `Dockerfile` in the tests folder. I think you mean the one in the repo's root folder.
- Note that the latest version of RV is v0.30. 
- See the [`docker/build` script](https://github.com/azavea/raster-vision/blob/master/docker/build) for building the docker image. It has values for the arguments used in the `Dockerfile`.
- `TARGETARCH` and `TARGETPLATFORM` are set by docker at build time.
